src/NewPipelinePage.tsx

```diff
@@ -213,7 +213,11 @@ const NewPipelinePage = (): React.ReactElement => {
                 />
               </Form.Group>
               <CommandRow>
-                <Button variant="primary" onClick={sourceConversionHandler}>
+                <Button
+                  variant="primary"
+                  disabled={!astarte.token?.can('flow', 'GET', '/pipelines')}
+                  onClick={sourceConversionHandler}
+                >
```

Comment:
We can avoid this check and let the user review the code for the draft pipeline.

What we need to do though, is to hide the "Create new pipeline" button when the permission is missing:

```
!astarte.token?.can('flow', 'POST', '/pipelines')
```